src/index.js

```diff
@@ -61,126 +66,197 @@ const monitorChromecastPlayer = function (player, options) {
       player_is_paused: isPaused,
       video_source_width: videoSourceWidth,
       video_source_height: videoSourceHeight,
-
+      video_title: title,
       player_autoplay_on: autoplay,
       player_preload_on: undefined,
       video_source_url: mediaUrl,
       video_source_mime_type: contentType,
       video_source_duration: duration,
-
+      viewer_device_name: 'Chromecast',
       video_poster_url: postUrl,
       player_language_code: undefined
     };
   };
 
   player.muxListener = function (event) {
     log.info('MuxCast: event ', event);
-    switch (event.type) {
-      case cast.framework.events.EventType.REQUEST_LOAD:
-        if (event.requestData.media !== undefined) {
-          if (event.requestData.media.contentId !== undefined) {
-            mediaUrl = event.requestData.media.contentId;
-          }
+    if (inAdBreak === false) {
```

Comment:
I'd just say `if (!inAdBreak) {`, in general comparing to `true`/`false` is a little bit unnecessary (javascript, of course, makes this more tricky with their whole truthy/falsy thing, but in general this isn't necessary)